docs/en/notes/faq.md

```diff
@@ -26,6 +26,11 @@ We list some common troubles faced by many users and their corresponding solutio
   1. Uninstall existing `mmcv-lite` in the environment using `pip uninstall mmcv-lite`.
   2. Install `mmcv` following the [installation instruction](https://mmcv.readthedocs.io/en/2.x/get_started/installation.html).
 
+- "Microsoft Visual C++ 14.0 or graeter is required" during installation on Windows.
+
+  The reason is that the corresponding C ++ environment compilation dependencies are lacking, and you need to download it at Microsoft officials [visual-cpp-build-tools](https://visualstudio.microsoft.com/zh-hans/visual-cpp-build-tools/),
```

Comment:
The reason is that there lacks corresponding C ++ compilation dependencies in the environment and you need to